governance/using.rst

```diff
@@ -228,6 +228,113 @@ discussion with the community. The steps to be taken are almost
+the Dash budget proposal generator, but it requires several additional steps to
+manually construct the proposal governance object.
+
+Assemble the proposal data
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To prepare a proposal, put the proposal details such as name and payout address
+into a JSON object similar to the example shown below. 
+
+.. code-block:: json
+
+  {
+    "name": "Test-proposal_1",
+    "payment_address": "yd5KMREs3GLMe6mTJYr3YrH1juwNwrFCfB",
+    "payment_amount": 10,
+    "url": "https://www.mydashtestproposal.com",
+    "start_epoch": 1635750000,
+    "end_epoch": 1636750000,
+    "type": 1
+  }  
+
+Set the ``type`` field to ``1`` for all proposals.
+
+The ``start_epoch`` and ``end_epoch`` fields are Unix epoch timestamps
+indicating the time range in which the proposal can receive payments. Typically
+you will set the ``start_epoch`` to approximately halfway between the superblock
+where payment is first desired and the preceding one. Set ``end_epoch`` to
+approximately 2 weeks after the superblock where the final payment is desired.
+You can use a site like https://www.epochconverter.com/ to convert the start and
+end dates to the epoch values for these fields.
+
+Serialize the proposal data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The proposal information must be serialized to hex before it can be submitted to
+the network. Remove all spaces from the JSON object::
+
+  {"name":"Test-proposal_1","payment_address":"yd5KMREs3GLMe6mTJYr3YrH1juwNwrFCfB","payment_amount":10,"type":1,"url":"http://test.com","start_epoch":1635750000,"end_epoch":1636750000}
+
+Convert the resulting JSON to its hex equivalent. Sites like
+https://codebeautify.org/string-hex-converter provide an easy way to do this::
+
+  7b226e616d65223a22546573742d70726f706f73616c5f31222c227061796d656e745f61646472657373223a227964354b4d52457333474c4d65366d544a597233597248316a75774e777246436642222c227061796d656e745f616d6f756e74223a31302c2274797065223a312c2275726c223a22687474703a2f2f746573742e636f6d222c2273746172745f65706f6368223a313633353735303030302c22656e645f65706f6368223a313633363735303030307d
+
+Prepare the collateral transaction
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Finally, open your Dash Core wallet console and use the ``gobject prepare``
+command to complete the proposal preparation and submit the collateral
+transaction. See the `Core developer documentation
+<https://dashcore.readme.io/docs/core-api-ref-remote-procedure-calls-dash#gobject-prepare>`_
+for additional details.
+
+.. warning::
+  Running this command will create a transaction spending 5 DASH from the wallet
+  as collateral for the proposal being created. Only run this command once you
+  have verified all the proposal information. The transaction is not reversible
+  once sent.
+  
+::
+
+  gobject prepare <parent-hash> <revision> <time> <data-hex>
+
+- ``parent-hash`` - set to ``0``
+- ``revision`` - set to ``1``
+- ``time`` - set to the current Unix epoch time
```

Comment:
How accurate does this time have to be?

Reply:
Good question. I'm consulting an expert :mage:

Reply:
@strophy adjusted the wording slightly. Per my attempt on testnet, the time doesn't technically have to be anything recent. I used a nice round epoch time that turned out to be a date in 2017. The proposal still showed up fine on the network when I submitted it.